### PR TITLE
Add NPC examines

### PR DESCRIPTION
**Pull request by asenechal**
## What has been done?

- Updated npcs.yml with more relevant npc examine text. Pulled from higher revision released source and open source projects. This is not a complete list, but at least something to work with.
- Defined opcode 92, added logic to appropriate handler class and updated the message decoder set.
- Added npc metadata service to game config file.

## Has your code been documented?
There are not many actual lines of code here, so inline comments were not added as it's fairly straightforward. If required I can add, but the small blocks of code here are in line with OpNpc 1-5 handler classes.

### CODE REVIEW

File: data/cfg/npcs.yml
Comment (HarleyGilpin) [inline data/cfg/npcs.yml:1]: NPC examine definitions look great, thanks for the nice addition!

File: data/packets.yml
Comment (HarleyGilpin) [inline data/packets.yml:1]: Have you tested to make sure the opcode is correct? My 667 opcode table says that opcode 92 is quickchat related.

Comment (HarleyGilpin) [reply data/packets.yml:1]: Also should the name be movement_type instead of unused?

**Review by HarleyGilpin — CHANGES_REQUESTED** [review]
Please look over the comments, thanks for the contribution and great work overall.

**Review by HarleyGilpin — APPROVED** [review]
Approved

File: data/packets.yml
Comment (asenechal) [reply data/packets.yml:1]: Hey, yes this is all tested. When examining npc's with debugging there was previously an unhandled opcode error for 92. I'm positive I checked on another 667 or R-S to confirm. 

While working through getting this implemented I didn't pick up on 'movement_type' as being a name for buffer movement. I assumed it was some left over copy/paste... but makes more sense looking at it a few days later. Can change in next PR to keep things consistent.

**Pull request merged**